packages/stellar-axelar-std/src/testdata/panics_when_golden_file_does_not_exist.golden

```diff
@@ -1 +0,0 @@
-something
```

Comment:
why wasn't this caught in the original PR? stale golden files should cause test to fail
https://github.com/axelarnetwork/axelar-cgp-stellar/blob/root-auth-upgrader/.github/workflows/test.yaml#L42